### PR TITLE
Don't change timestamp when splitting a way

### PR DESCRIPTION
**Pull request by Helium314**
Fixes resurvey quests disappearing when splitting a way (only with auto-sync off), as discussed in #3567.
Tested and works.

### CODE REVIEW

File: app/src/main/java/de/westnordost/streetcomplete/data/osm/edits/split_way/SplitWayAction.kt [hunk 1]
```diff
@@ -131,9 +131,10 @@ private fun getSplitWayAtIndices(
 
     return nodesChunks.mapIndexed { index, nodes ->
         if (index == indexOfChunkToKeep) {
-            Way(originalWay.id, nodes, tags, originalWay.version, currentTimeMillis())
+            // keep the original timestampEdited, so resurvey quests are still shown after splitting
```
Comment (matkoniecz) [inline app/src/main/java/de/westnordost/streetcomplete/data/osm/edits/split_way/SplitWayAction.kt:134]: as it refers to both Way creations, maybe keep it one level higher - before `if (index == indexOfChunkToKeep)`?

**Comment by matkoniecz** [comment]
Also, is it working only with auto-sync disabled? In such case maybe also mention it in the comment.

File: app/src/main/java/de/westnordost/streetcomplete/data/osm/edits/split_way/SplitWayAction.kt (hunk 1 above)
Comment (westnordost) [reply app/src/main/java/de/westnordost/streetcomplete/data/osm/edits/split_way/SplitWayAction.kt:134]: The comment? makes sense

**Comment by Helium314** [comment]
Makes sense, both done

**Pull request merged**